code/modules/surgery/other.dm

```diff
@@ -229,12 +240,15 @@
 	if(!valid_container)
 		return FALSE
 
-	if(container.reagents.has_reagent(/decl/material/liquid/antiseptic))
-		return TRUE
+	if(!container.reagents?.total_volume)
+		return FALSE
 
-	for(var/rtype in container?.reagents?.reagent_volumes)
-		var/decl/material/liquid/ethanol/booze = GET_DECL(rtype)
-		if(istype(booze) && booze.strength <= 40)
-			return TRUE
+	// Must be impure.
```

Comment:
```suggestion
	// This check means it's impure.
```
the way this is phrased makes it sound like we're saying that the reagents "must be impure," as in it's a requirement to proceed. i guess you don't need to change it, but like, still

Reply:
i did say "I guess you don't need to change it," i guess maybe i should've been clearer and said "non-blocking" explicitly

also ironic to have this miscommunication when i was complaining about the phrasing of a comment being unclear. whoops. should remove the log from my own eye before criticizing the speck in my neighbor's etc